src/vs/workbench/contrib/terminal/browser/terminalMainContribution.ts

```diff
@@ -79,13 +70,6 @@ export class TerminalMainContribution extends Disposable implements IWorkbenchCo
 				separator: ''
 			}
 		});
-
-		// Register log channel
-		this._registerLogChannel('ptyHostLog', localize('ptyHost', "Pty Host"), URI.file(join(environmentService.logsPath, `${TerminalLogConstants.FileName}.log`)));
 	}
 
-	private _registerLogChannel(id: string, label: string, file: URI): void {
-		const promise = registerLogChannel(id, label, file, this._fileService, this._logService);
-		this._register(toDisposable(() => promise.cancel()));
-	}
 }
```

Comment:
@Tyriar you do not need to create output channel now. You can include all necessary info while creating the logger and system will create it automatically.